docs/config/README.md

```diff
@@ -292,6 +292,27 @@ style = "bold blue"
 symbol = "🅰 "
 ```
 
+## AZURE
+
+The `azure` module shows the current Azure Subscription. This is based on 
+`~/.azure/clouds.config` and `~/.azure/azureProfile.json` files.
+
+### Options
+
+| Variable          | Default         | Description                                                                 |
+| ----------------- | --------------- | --------------------------------------------------------------------------- |
+| `format`          | `"on ☁️ [$subscription](blue bold) "`         | The format for the Azure module to render in                  |
```

Comment:
With the other modules we have been keeping the symbol and style as separate variables to make it easier for users to change just the symbol or style.